tests/e2e/ctl_v2_test.go

```diff
@@ -289,6 +289,7 @@ func testCtlV2Backup(t *testing.T, snapCount int, v3 bool) {
 
 	if v3 {
 		// v3 must lock the db to backup, so stop process
+		time.Sleep(time.Second)
```

Comment:
need comment on why

Reply:
Removed in current pr since we keep using v2 publish(). It seems like the cluster needs more time publish using publishV3 which makes `TestCtlV2BackupV3` a flaky test. Adding a sleep is to address this.